taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java

```diff
@@ -53,7 +55,7 @@
             camundaSystemUrl, CamundaSystemConnectorImpl.URL_GET_CAMUNDA_FINISHED_EVENTS);
 
     List<ReferencedTask> referencedTasks =
-        getReferencedTasksFromCamundaTaskEvents(camundaTaskEvents);
+        getReferencedTasksFromCamundaTaskEvents(camundaTaskEvents, null);
```

Comment:
why do we need 'null' here?

Reply:
This is within the retrievedFinishedCamundaTasks method which doesn't need an identifier because it doesn't matter which SystemConnector fetches/processes it. getReferencedTasksFromCamundaTaskEvents method is a common helper method which is also used by retrieveNewStartedCamundaTasks...but this needs the system identifier.

Reply:
Wouldn't it still be more consistent to use the identifier even when we technically don't need them? Why not explicitly stick to the fact that all events are processed by the corresponding connector?